src/test/java/com/conveyal/datatools/DataSanitizerTest.java

```diff
@@ -109,4 +124,31 @@ void canIdentifyOrphanedDBSchemas() {
     void canRemoveOrphanedDBSchema() {
         assertEquals(1, DataSanitizer.deleteOrphanedDBSchemas(Set.of(orphanedDBSchema)));
     }
+
+    @Test
+    void canAuditFeedVersions() {
+        List<DataSanitizer.FeedVersionAudit> result = DataSanitizer.feedVersionAudit();
+        assertTrue(result.stream().anyMatch(audit -> audit.projectName.equals(project.name)));
+        assertTrue(result.stream().anyMatch(audit -> audit.feedSourceName.equals(feedSourceParent.name)));
+        assertTrue(result.stream().anyMatch(audit -> audit.feedSourceName.equals(feedSourceWithObsoleteFeedVersion.name)));
+    }
+
+    @Test
+    void canDeleteObsoleteFeedVersions() {
+        // From the initial four feed versions, delete the older two.
```

Comment:
IMO, the wording below better reflects the logic, if I understood it correctly.
```suggestion
        // From the initial four feed versions, keep the two most recent.
```

Reply:
Nit - The test would be easier to read with different numbers of deleted vs. kept versions.